parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/xcm_config.rs

```diff
@@ -96,17 +96,16 @@ pub type XcmOriginToTransactDispatchOrigin = (
 	// transaction from the Root origin.
 	ParentAsSuperuser<RuntimeOrigin>,
 	// Native signed account converter; this just converts an `AccountId32` origin into a normal
-	// `Origin::Signed` origin of the same 32-byte value.
+	// `RuntimeOrigin::Signed` origin of the same 32-byte value.
 	SignedAccountId32AsNative<RelayNetwork, RuntimeOrigin>,
 	// Xcm origins can be represented natively under the Xcm pallet's Xcm origin.
 	XcmPassthrough<RuntimeOrigin>,
 );
 
 match_types! {
-	// TODO: map gov2 origins here - after merge https://github.com/paritytech/cumulus/pull/1895
-	pub type ParentOrParentsExecutivePlurality: impl Contains<MultiLocation> = {
+	pub type ParentOrParentsPlurality: impl Contains<MultiLocation> = {
```

Comment:
Nit: `ParentOrParentsPlurality` and `ParentOrSiblings` seem to be duplicated for all the bridge hubs, and maybe for other runtimes as well. Could we define them once in a common place instead and reuse them ?

Reply:
@joepetrowski @gilescope @muharem 
good point, `ParentOrParentsPlurality / ParentOrSiblings` looks the same everywhere now and probably wont change ever (maybe we will add another ones to the tuple like `FellowsPlurality`),

wdyt about moving `ParentOrParentsPlurality / ParentOrSiblings` to the `parachains-common`?

Reply:
In this case it should in some specialized module/file which will be under the same review/merge rules.
I am personally fine with such duplications.

Reply:
Yeah, same